xdg/openshot-qt.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>openshot-qt.desktop</id>
```

Comment:
I guess using `<id/>` to associate the AppData with the application's `.desktop` file [is also deprecated](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#spec-appdata-filespec). (OpenShot's is `openshot-qt.desktop`, BTW.) According to that spec `<id/>` is now supposed to be a reverse-DNS-style identifier, and `<launchable/>` associates the AppData with a `.desktop` file. So, if I'm interpreting correctly, I'd suggest:
```xml
<id>org.OpenShot.openshot-qt</id>
<launchable type="desktop-id">openshot-qt.desktop</launchable>
```
and while we're at it, also
```xml
<provides>
  <binary>openshot-qt</binary>
</provides>
```

Reply:
Oh, and while I'm suggesting fields to add, one more:
```xml
<url type="bugtracker">https://github.com/OpenShot/openshot-qt/issues</url>
```
Thanks!